samples/matter/lock/CMakeLists.txt

```diff
@@ -17,9 +17,8 @@ if(CONF_FILE)
 endif()
 
 if(CONF_FILE_NAME STREQUAL "prj_thread_wifi_switched.conf")
-    set(PM_STATIC_YML_FILE ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static_dfu_wifi_ext_patch.yml)
-elseif(NOT CONF_FILE_NAME STREQUAL "prj_no_dfu.conf")
-    set(PM_STATIC_YML_FILE ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static_dfu.yml)
+    # TODO: Add Kconfig for adding custom Matter compilation flags
```

Comment:
I believe It's not really needed now that LTO is globally enabled for this variant.